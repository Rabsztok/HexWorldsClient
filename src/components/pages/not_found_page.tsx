import React from 'react'

const NotFoundPage = () => (
  <div>
    <h2>Uh oh!</h2>
    <p>We could not find what you were looking for!</p>
  </div>
)

export { NotFoundPage }
export default NotFoundPage
