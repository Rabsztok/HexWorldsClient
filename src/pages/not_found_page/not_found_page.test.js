import React from 'react'
import { shallow } from 'enzyme'
import { NotFoundPage } from './not_found_page'

describe('NotFoundPage', () => {
  it('renders without errors', () => {
    const component = shallow(<NotFoundPage />)
    expect(component).toMatchSnapshot()
  })
})
