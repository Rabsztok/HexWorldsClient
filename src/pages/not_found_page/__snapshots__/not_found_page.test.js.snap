// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders without errors 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Uh oh!
</h2>,
        <p>
          We could not find what you were looking for!
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Uh oh!",
        },
        "ref": null,
        "rendered": "Uh oh!",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "We could not find what you were looking for!",
        },
        "ref": null,
        "rendered": "We could not find what you were looking for!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Uh oh!
</h2>,
          <p>
            We could not find what you were looking for!
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Uh oh!",
          },
          "ref": null,
          "rendered": "Uh oh!",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "We could not find what you were looking for!",
          },
          "ref": null,
          "rendered": "We could not find what you were looking for!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
